typescript/rest-express-auth/.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
+dist/
+*.env*
```

Comment:
The `prisma-examples` repository includes a `.gitignore` file at the root of the project. This won't be necessary 🙂